docs/api/apiv3/paths/days_week.yml

```diff
@@ -0,0 +1,100 @@
+# /api/v3/days/week
+---
+get:
+  summary: Lists week days
+  operationId: list_week_days
+  tags:
+    - Work Schedule
+    - Non working days
+    - Week days
+  description: |-
+    Lists week days with work schedule information.
+  responses:
+    '200':
+      description: OK
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/week_day_collection_model.yml'
+    '400':
+      description: Returned if the client sends invalid request parameters
```

Comment:
This endpoint should be publicly available, so no 403.
If we restrict to logged in users, that means frontend will not be able to display elements of public projects properly (Gantt chart, calendars, etc.) because the week days and non-working days information can not be fetched.